docs/docs/api/promise.on.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+id: promise.on
+title: Promise.on
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.on
+
+```js
+Promise.on(String eventName("created"|"chained"|"fulfilled"|"rejected"|"following"|"cancelled"), Function handler, [Boolean simpleEventAPI]) -> undefined
```

Comment:
@spion Exactly this is what I mean: why would you be interested when some promise is created in dbLibrary? I think in most cases this is not important information, unless you want to debug the library you are using. Also it makes this events not very consistent: lets say dbLibrary uses bluebird and dbLibraryUtilityLibrary uses Q then the Q lifecycle events will not be reported. 
@petkaantonov For performance reasons I have to attach event handlers to functions inside single bluebird instance. This means that to maintain events on global level I would need a list of all bluebird instances that exist, is this currently possible to get? Would it be acceptable I add  an array with bluebird instances to (process|window)?